tests/Aspire.Hosting.Tests/WithEndpointTests.cs

```diff
@@ -459,7 +459,8 @@ public async Task VerifyManifestProjectWithDefaultHttpEndpointsDoesNotAllocatePo
             .WithHttpEndpoint(name: "hp")       // Won't get targetPort since it's the first http
             .WithHttpEndpoint(name: "hp2")      // Will get a targetPort
             .WithHttpsEndpoint(name: "hps")     // Won't get targetPort since it's the first https
-            .WithHttpsEndpoint(name: "hps2");   // Will get a targetPort
+            .WithHttpsEndpoint(name: "hps2")   // Will get a targetPort
+            .WithEndpoint(scheme: "tcp", name: "tcp0");  // Will get a targetPort
```

Comment:
super nit: maybe an additional `.WithEndpoint(scheme: "tcp", name: "tcp1")` to ensure that it works for a second tcp endpoint too.